parser/tools/generate_ast.cc

```diff
@@ -375,6 +483,60 @@ void emitNodeClassfile(ostream &out, NodeDef &node) {
     out << "    return to_string(buf);\n";
     out << "  }" << '\n';
     out << '\n';
+
+    // toRuby
+    out << "  std::string " << node.name << "::toRuby(const core::GlobalState &gs, int tabs) {" << '\n'
+        << "    fmt::memory_buffer buf;" << '\n';
+    out << "    fmt::format_to(buf, \"s(:" << rubyNodeNameForInternalNodeName(node.name) << "\");" << '\n';
```

Comment:
If we want it to be the case that this lookup always succeeds, why not just update `NodeDef` to have a third field which is the whitequark name? i.e., we'd have something like this:

```cpp
struct NodeDef {
    string name;
    string new_thing;
    vector<FieldDef> fields;
};

NodeDef nodes[] = {
    // alias bar foo
    {"Alias", "alias", vector<FieldDef>({{"from", Node}, {"to", Node}})},
    // ...
};

... << node.new_thing << ...
```

Reply:
I wanted to do it initially, but it feels dirty to me, and so I decided to extract it into external mapping. Different libraries/people may want to call nodes as they want, but that doesn't mean that all of this metadata is a part of the core logic.